boards/mikroe/quail/doc/mikroe_quail.rst

```diff
@@ -0,0 +1,99 @@
+.. zephyr:board:: mikroe_quail
+
+Overview
+********
+MikroE Quail for STM32 is a development board containing an `STM32F427`_
+microcontroller. It is equipped with four mikrobus sockets.
+The edges of the board are lined with screw terminals and USB ports for
+additional connectivity.
+
+Hardware
+********
+The Quail board contains the following connections:
+
+  - Four MikroBUS connectors
+  - lined with 32 screw terminals and USB ports for
+
+Furthermore the board contains three LEDs that are connected
+to the microcontroller.
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Connections and IOs
+===================
+
+The four mikroBUS interfaces are aliased in the device tree so that their
+peripherals can be accessed using ``mikrobus_N_INTERFACE`` so e.g. the spi on
+bus 2 can be found by the alias ``mikrobus_2_spi``. The counting corresponds
+with the marking on the board.
+
+For connections on the edge connectors, please refer to `Quail for STM32 User Manual`_.
+
+Programming and Debugging
+*************************
+
+.. zephyr:board-supported-runners::
+
+Applications for the ``mikroe_quail`` board can be built and flashed in the usual way
+(see :ref:`build_an_application` and :ref:`application_run` for more details).
+
+
+Flashing
+========
+The initial state of the board is set to lock.
```

Comment:
nit: this reads better as "The board ships with a locked flash, and will fail with the message:"